Resources/Locale/en-US/job/job-names.ftl

```diff
@@ -69,6 +73,8 @@ JobCargoTechnician = cargo technician
 JobCentralCommandOfficial = central command official
 JobChaplain = chaplain
 JobChef = chef
+JobSAI = station AI
+JobChiefServiceSupervisor = chief service supervisor
```

Comment:
These should go in the SimpleStation FTL folder.
Should be Resources/Locale/en-US/SimpleStation14/ if I'm not mistaken.